README.md

```diff
@@ -6,7 +6,7 @@ To access charts from this from the cli repository add it:
 
 ```sh
 
-helm repo add rh-dse https://rh-datascience-and-edge-practice.github.io/helm-charts/
+helm repo add rh-ai-svcs https://redhat-ai-services.github.io/helm-charts/
```

Comment:
What do you think of `redhat-ai-services` instead of `rh-ai-svcs`?  That is what I have in other places in the repo.

Reply:
yep, I'm good with that.